frontend/src/components/Home/MainDrawer.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import Box from '@mui/material/Box';
+import Drawer from '@mui/material/Drawer';
+import List from '@mui/material/List';
+import Divider from '@mui/material/Divider';
+import ListItem from '@mui/material/ListItem';
+import IconButton from '@mui/material/IconButton';
+import MenuIcon from '@mui/icons-material/Menu';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import TheaterComedyIcon from '@mui/icons-material/TheaterComedy';
+import VolumeUpIcon from '@mui/icons-material/VolumeUp';
+import ImageIcon from '@mui/icons-material/Image';
+import { Typography } from '@mui/material';
+
+class MainDrawer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            open: false,
+        };
+    }
+
+    _handleClickOpen = () => {
+        this.setState({ open: true });
+    };
+
+    _handleClose = () => {
+        this.setState({ open: false });
+    };
+
+    render() {
+        return (
+            <div>
+            <IconButton
+            size="large"
+            edge="start"
+            color="inherit"
+            aria-label="menu"
+            sx={{ mr: 2 }}
+            onClick={this._handleClickOpen}
+          >
+            <MenuIcon />
+          </IconButton>
+            <Drawer
+            anchor='left'
+            open={this.state.open}
+            onClose={this._handleClose}
+            >
+                <Box sx={{ width: 250 }}>
+                    <Typography sx={{ ...styles.title }}>Eelbot Config 🌊</Typography>
+                    <Divider/>
+                    <List>
+                        <ListItem button key={'Jokes'}>
```

Comment:
button is a [deprecated prop](https://mui.com/material-ui/api/list-item/#props), IIUC should switch these to be ```ListItemButton``` instead

Reply:
Really? that's interesting cause that prop was used in this example: https://mui.com/material-ui/react-drawer/

Reply:
Now it is resolved. see: https://github.com/emseers/eelbot-ui/commit/6aedc61523c23a498e5ecd7cd2624a6a2fb1c386